assets/src/application/ui/display/icons/svgs/appstoreFilled.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="64 64 896 896" focusable="false" class="ee-svg" data-icon="appstore" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M864 144H560c-8.8 0-16 7.2-16 16v304c0 8.8 7.2 16 16 16h304c8.8 0 16-7.2 16-16V160c0-8.8-7.2-16-16-16zm0 400H560c-8.8 0-16 7.2-16 16v304c0 8.8 7.2 16 16 16h304c8.8 0 16-7.2 16-16V560c0-8.8-7.2-16-16-16zM464 144H160c-8.8 0-16 7.2-16 16v304c0 8.8 7.2 16 16 16h304c8.8 0 16-7.2 16-16V160c0-8.8-7.2-16-16-16zm0 400H160c-8.8 0-16 7.2-16 16v304c0 8.8 7.2 16 16 16h304c8.8 0 16-7.2 16-16V560c0-8.8-7.2-16-16-16z"></path></svg>
```

Comment:
noticed that there are no new lines at the end of these svg files... is that for a technical reason?
i guess Prettier and the linters will ignore this because of the extension, so likely doesn't matter

Reply:
I was expecting prettier to do it's job out of the box, but had to add this plugin: https://github.com/prettier/plugin-xml